es_filebrowser_index.py

```diff
@@ -1,225 +0,0 @@
-#indexer_es_fb.py
```

Comment:
what happened to `es_filebrowser_index.py`? what is the reason you took it out?

Reply:
It got moved to a repo called `dcc-metadata-indexer`. It was used to generate the file oriented index. It was initially put here but then it made sense to concentrate all indexing code into the `dcc-metadata-indexer` repo.